website/templates/components/dashboard_templates.mako

```diff
@@ -153,7 +153,7 @@
             <br />
             <label>Template (Optional)</label>
             <span class="help-block">Start typing to search. Selecting project as template will duplicate its structure in the new project without importing the content of that project.</span>
-            <input type="hidden" class="select2-container createNodeTemplates" style="width: 100%">
+            <input type="hidden" class="select2-container createNodeTemplates" id="createNodeTemplatesInput" style="width: 100%">
```

Comment:
Ahh, gotchya. In that case, change the class name to meet our style guide: `create-node-templates`.